src/celmodel/mesh.cpp

```diff
@@ -88,18 +89,17 @@ Mesh::VertexDescription::operator=(const Mesh::VertexDescription& desc)
 // TODO: This should be called in the constructor; we should start using
 // exceptions in Celestia.
 bool
-Mesh::VertexDescription::validate() const
+VertexDescription::validate() const
 {
     for (unsigned int i = 0; i < nAttributes; i++)
     {
         VertexAttribute& attr = attributes[i];
 
         // Validate the attribute
-        if (attr.semantic >= SemanticMax || attr.format >= FormatMax)
-            return false;
-        if (attr.offset % 4 != 0)
-            return false;
-        if (attr.offset + VertexAttributeFormatSizes[attr.format] > stride)
+        if (attr.semantic >= VertexAttributeSemantic::SemanticMax
+            || attr.format >= VertexAttributeFormat::FormatMax
```

Comment:
with `enum class` these 2 lines are always false unless somebody wants to play with casts (but if somebody wants to show their feet who we are to forbid them?)